tst/regression/run_test.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+#========================================================================================
+# Athena++ astrophysical MHD code
+# Copyright(C) 2014 James M. Stone <jmstone@princeton.edu> and other code contributors
+# Licensed under the 3-clause BSD License, see LICENSE file for details
+#========================================================================================
+# (C) (or copyright) 2020. Triad National Security, LLC. All rights reserved.
+#
+# This program was produced under U.S. Government contract 89233218CNA000001 for Los
+# Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+# for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+# in the program are reserved by Triad National Security, LLC, and the U.S. Department
+# of Energy/National Nuclear Security Administration. The Government is granted for
+# itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works, distribute copies to
+# the public, perform publicly and display publicly, and to permit others to do so.
+#========================================================================================
+
+# Python modules
+import argparse
+import os
+import sys
+
+""" To prevent littering up imported folders with .pyc files"""
+sys.dont_write_bytecode = True
+
+# Parthenon modules
+import utils.test_case as tc 
+
+""" Split a path into folders """
+def SplitPathIntoFolders(path):
+    folders = []
+    while 1:
+        path, folder = os.path.split(path)
+        if folder != "":
+            folders.append(folder)
+        else:
+            if path != "":
+                folders.append(path)
+            break
+    return folders
+
+def checkRunScriptLocation(run_test_py_path):
+  
+    """ Check that run_test is in the correct folder """
+    folders = SplitPathIntoFolders(run_test_py_path)
+    if( os.path.join(folders[2], folders[1], folders[0]) != "parthenon/tst/regression" ): 
```

Comment:
Hmm. I see now why you wrote `SplitPathIntoFolders`. I think Python has built in functionality for this. If I recall, it's something like:
```python
if not os.path.normpath(run_test_py_path).startswith(os.path.normpath("parthenon/tst/regression"))
```
The normpaths might be unnecessary, but `run_test_py_path` does need to be a path object, not a string.

Reply:
This is an improvement, thanks!